package.json

```diff
@@ -19,6 +19,9 @@
   "bugs": {
     "url": "https://github.com/nuxt-modules/sitemap/issues"
   },
+  "engines": {
+    "node": ">=18.0.0"
```

Comment:
### Note

The Node version matches the Nuxt CLI.

- https://github.com/nuxt/cli/blob/7657a7cc4d4f30deaf7e8ce04721299fd187a442/src/utils/engines.ts#L7